projects/netgrif-components-core/src/lib/task/services/task-data.service.ts

```diff
@@ -428,7 +430,7 @@ export class TaskDataService extends TaskHandlingService implements OnDestroy {
      * @param body hold the data that was sent in request
      */
     protected processSuccessfulSetDataRequest(setTaskId: string,
```

Comment:
if the method is called `processSuccessfulSetDataRequest` it should only do that and it should not check if the response was indeed a success. That should be resolved elsewhere

Reply:
Done